tests/test_boot.py

```diff
@@ -57,6 +57,9 @@ def test_boot(format: OutputFormat) -> None:
 
         image.qemu(options=options)
 
+        if format in (OutputFormat.disk, OutputFormat.directory):
+            image.vmspawn(options=options)
```

Comment:
We should skip this if vmspawn is not installed since it won't be available on many systems for a long time. You can use find_binary() and systemd_tool_version() to check if it's available and to check if the version > 256.

You should also extend the check_tools() function with the same check.

Reply:
914d862 now contains these checks